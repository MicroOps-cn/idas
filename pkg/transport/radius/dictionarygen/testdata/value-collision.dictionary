ATTRIBUTE DSL-Transmission-System 100 integer
VALUE     DSL-Transmission-System         ADSL2           2
VALUE     DSL-Transmission-System         ADSL2+          3
