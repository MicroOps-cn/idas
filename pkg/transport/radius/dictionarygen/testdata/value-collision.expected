Constants:
	const (
		DSLTransmissionSystem_Type radius.Type = 100
	)
Types:
	type DSLTransmissionSystem uint32
		const (
			DSLTransmissionSystem_Value_ADSL2	DSLTransmissionSystem	= 2
			DSLTransmissionSystem_Value_ADSL2Plus	DSLTransmissionSystem	= 3
		)
		func (a DSLTransmissionSystem) String() string
Functions:
		func DSLTransmissionSystem_Add(p *radius.Packet, value DSLTransmissionSystem) (err error)
		func DSLTransmissionSystem_Del(p *radius.Packet)
		func DSLTransmissionSystem_Set(p *radius.Packet, value DSLTransmissionSystem) (err error)
